src/main/java/pvpmode/internal/server/utils/PvPServerUtilsProvider.java

```diff
@@ -38,6 +37,18 @@ public EntityPlayerMP getPlayer (String name)
         return server.getServerConfigurationManager ().func_152612_a (name);
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public EntityPlayerMP getPlayer (UUID uuid)
+    {
+        for (EntityPlayerMP player : (List<EntityPlayerMP>) server.getServerConfigurationManager ().playerEntityList)
+        {
+            if (player.getUniqueID ().equals (uuid))
+                return player;
+        }
+        return null;
```

Comment:
As of Java 8 the Stream API is the preferred way to do things like this.
```suggestion
        return ((List<EntityPlayerMP>) server.getServerConfigurationManager ().playerEntityList)
                .stream ().filter (player -> player.getUniqueID ().equals (uuid)).findFirst ().orElse (null);
```

Reply:
Indeed. I switch between the Stream API and the for loop somewhat inconsistent...